Changelog.md

```diff
@@ -8,6 +8,7 @@
 - Added all the `filter`/`map` operators that were available for `Observable<Response>` to `Single<Response>` as well.
 - Added `AuthorizationType` to `AccessTokenAuthorizable` representing request headers of `.none`, `.basic`, and `.bearer`. 
 - Added tests for `Single<Response>` operators.
+- Be notified on progress callback also for data request.
```

Comment:
Can we rather stick to convention "Added/Fixed"? e.g. "Fixed a bug with no progress for data request"